tools/c7n_tencentcloud/c7n_tencentcloud/client.py

```diff
@@ -31,6 +34,70 @@ def retry_result(resp):
     return False
 
 
+def profile_handel(
+        profile: str,
+        cred_path=os.path.join(os.path.expanduser('~'), '.tencentcloud/credentials')
+) -> Union[STSAssumeRoleCredential, Credential]:
+    """
+        params:
+            profile: the profile name
+            profile_path: the profile path, deafult path is '~/tencentcloud/credentials'
+        des:
+            support use profile to auth account and multi-account
+        credentials details:
+            ```
+                [default] (if 'source_profile' is default, this is required!)
+                secret_id:xxxx
+                secret_key:xxxx
+                [profile]
+                role_arn: xxx (required)
+                session_name: xxx (default is 'custodian-job')
+                duration_seconds: 3600 (default is 3600)
+                source_profile: xxx (required, must be in ('default', 'cvm_metadata'))
+            ```
+            parms:
+                source_profile: the auth method
+                    default: use ak/sk to assume role
+                        secret_id: secret_id
+                        secket_key: secket_key
+                    cvm_metadata: use cvm role to assume role
+    """
+    if not os.path.exists(cred_path):
+        raise TencentCloudSDKException(f'not find the cred path by "{cred_path}"')
```

Comment:
there are a few context where this path won't exist and this should exit gracefully